referee.c

```diff
@@ -0,0 +1,180 @@
+#include <stdio.h>
+#include <time.h>
+#include <Windows.h>
+#include <stdlib.h>
+#include "referee.h"
+
+//#define N 3
+const char markerX = 'X';
+const char marker0 ='0';
+const char markerEmpty = ' ';
+
+
+int referee(int (*playerX)(char marker, char* board_copy), int (*player0)(char marker, char* board_copy) ){

```

Comment:
1. Please add a description to the function.
2. It's really better to have `typedef` for the type (bit not mandatory, though).

@SYurt are we gonna have function pointers to call algorithms or just indexes and a common function that calls algorithms like:
```c
int getMove(int algorithmIndex, int board[N*N], char marker) {
   switch (algorithmIndex) {
   case 1: return algorithm1(board, marker);
   case 2: return algorithm2(board, marker);
   ...
   }
}
```